projects/components/src/combo-box/combo-box.component.ts

```diff
@@ -85,6 +85,7 @@ import { ComboBoxMode, ComboBoxOption, ComboBoxResult } from './combo-box-api';
             *ngFor="let option of this.filteredOptions; index as i"
             [class.selected]="this.highlightedOptionIndex === i"
             (click)="this.onOptionClick(option)"
+            [htcTooltip]="option.tooltip ? option.tooltip : ''"
```

Comment:
Is the ternary needed? the tooltip directive should respond to undefined/null/empty string all in the same way

Reply:
You are probably right. I didn't even try. I'll change this locally.